src/poly/multiopen/prover.rs

```diff
@@ -110,8 +108,9 @@ impl<C: CurveAffine> Proof<C> {
 
         let (opening, q_evals) = loop {
             let mut transcript = transcript.clone();
-            let mut transcript_scalar = transcript_scalar.clone();
-            hash_point(&mut transcript, &f_commitment).unwrap();
+            transcript
+                .absorb_point(&f_commitment)
+                .map_err(|_| Error::SamplingError)?;
```

Comment:
Relative to the previous behaviour, we now absorb `f_commitment` before we squeeze the scalar hasher to absorb the scalars from the PLONK prover into the base hasher. I'm not sure if the reordering matters (i.e. whether we should squeeze the scalar hasher into the base hasher if it needs squeezing, when we call `absorb_point` as well as when we call `squeeze`).